docs/en/setup/service-agent/java-agent/TLS.md

```diff
@@ -25,9 +24,9 @@ agent.force_tls=${SW_AGENT_FORCE_TLS:true}
 
 ## Enable mutual TLS
 
-- Sharing gRPC server must be started with enabled mTLS. More details see `receiver-sharing-server` section in `application.yaml`. Please refer to [gRPC SSL](../../backend/grpc-ssl.md)  
+- Sharing gRPC server must be started with enabled mTLS. More details see `receiver-sharing-server` section in `application.yaml` of SkyWalking OAP Server.  
```

Comment:
What is `receiver-sharing-server` section for now? I think you should refer to the correct keyword? The keyword should be able to used here to search for documentation.

![image](https://user-images.githubusercontent.com/5441976/131251651-f7e8d19a-b7e2-4231-ae25-449f0e572112.png)

From https://github.com/apache/skywalking/pull/7565/files#diff-c6b2f69e272d64016aa069e5b2469b95aeabec06cea77c3f88b734dea84ff4cd, it should be `Security(SSL/TLS/mTLS)`.